From cc6dc1b7fc78edfd92472318ede0da2c1ee376f3 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 15 Dec 2017 09:02:34 +0900
Subject: [PATCH 138/909] clk: renesas: r8a7796: Add iVDP1C clock

commit a27e5149e08418e62bd7044cf2022163200641b9 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds iVDP1C clock to R8A7796 SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/renesas/r8a7796-cpg-mssr.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/renesas/r8a7796-cpg-mssr.c b/drivers/clk/renesas/r8a7796-cpg-mssr.c
index c854a2a5539d..6e1adcd7d37c 100644
--- a/drivers/clk/renesas/r8a7796-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a7796-cpg-mssr.c
@@ -122,6 +122,7 @@ static const struct cpg_core_clk r8a7796_core_clks[] __initconst = {
 static const struct mssr_mod_clk r8a7796_mod_clks[] __initconst = {
 	DEF_MOD("3dge",			 112,	R8A7796_CLK_ZG),
 	DEF_MOD("fdp1-0",		 119,	R8A7796_CLK_S0D1),
+	DEF_MOD("ivdp1c",		 128,	R8A7796_CLK_S0D2),
 	DEF_MOD("vcplf",		 130,	R8A7796_CLK_S0D2),
 	DEF_MOD("vdpb",			 131,	R8A7796_CLK_S0D2),
 	DEF_MOD("scif5",		 202,	R8A7796_CLK_S3D4),
-- 
2.17.1

